packages/terra-date-picker/tests/jest/DateInput.test.jsx

```diff
@@ -19,6 +19,7 @@ it('should render a default date input with all props', () => {
       onChange={() => {}}
       onClick={() => {}}
       onKeyDown={() => {}}
+      aria-label="Terra.datePicker.date"
```

Comment:
Does it throw accessibility error for not having Aria-Label here..?  if yes then change the value of aria-label to 'date' instead of Terra.datePicker.date.